python/lsst/ts/salobj/topics/remote_command.py

```diff
@@ -277,19 +276,23 @@ async def next_ackcmd(self, ackcmd, timeout=DEFAULT_TIMEOUT, wait_done=True):
         Parameters
         ----------
         ackcmd : `SalInfo.AckCmdType`
-            The command acknowledgement returned by
-            the previous wait (e.g. from `start`).
+            Command acknowledgement returned by the previous call to
+            `set_start`, `start` or `next_ackcmd`.
         timeout : `float`, optional
             Time limit, in seconds. If None then use ``DEFAULT_TIMEOUT``.
-            This time limit is for the entire command if wait_done
-            is true, else it is for the first acknowledgement
-            after the initial "CMD_ACK".
+            This time limit is for the entire command if ``wait_done``
```

Comment:
```suggestion
            This time limit is for the remainder of the command if ``wait_done``
```

Reply:
Good catch. "remainder" makes the line too long so I plan to use:
```
This time limit is for the command to finish, if ``wait_done``
is true, else it is for the next command acknowledgement.
```